Documentation/typo3cms/extensions/Index.rst

```diff
@@ -11,13 +11,22 @@ Extensions
 .. First
    You may add normal rst content here.
 
+Here you can find documentation for third party extensions
+available through :ref:`TYPO3 Extension Repository (TER) <https://extensions.typo3.org/>`.
+
+If you're looking for documentation of the extensions shipped with TYPO3 Core, go to the `system extensions section`. 
```

Comment:
Please check proper link definition.

Looks like there are two underscores missing at the end of the link. Compared to the examples "json file about extensions" and "json file about system extensions".